src/admin/components/forms/field-types/Checkbox/Input.tsx

```diff
@@ -43,9 +46,12 @@ export const CheckboxInput: React.FC<CheckboxInputProps> = (props) => {
         name={name}
         checked={checked}
         readOnly
+        tabIndex={-1}
       />
       <button
         type="button"
+        onFocus={() => setIsFocused(true)}
```

Comment:
Buttons already have the ability to be targeted when they are focused, we should use the css `:focus` to style this.